docs/en/sql-reference/statements/create/database.md

```diff
@@ -26,4 +26,33 @@ ClickHouse creates the `db_name` database on all the servers of a specified clus
 
 ### ENGINE {#engine}
 
-[MySQL](../../../engines/database-engines/mysql.md) allows you to retrieve data from the remote MySQL server. By default, ClickHouse uses its own [database engine](../../../engines/database-engines/index.md). There’s also a [lazy](../../../engines/database-engines/lazy.md) engine.
+[MySQL](../../../engines/database-engines/mysql.md) allows you to retrieve data from the remote MySQL server. By default, ClickHouse uses its own [database engine](../../../engines/database-engines/index.md). There is also a [lazy](../../../engines/database-engines/lazy.md) engine.
```

Comment:
It looks outdated, a long time ago we had only `Ordinary`, `MySQL` and `Lazy` engines, now we have much more and should not highlight only `MySQL` and `Lazy` here.
```suggestion
By default, ClickHouse uses its own [Atomic](../../../engines/database-engines/atomic.md) database engine. There are also [Lazy](../../../engines/database-engines/lazy.md), MySQL, PostgresSQL, MaterializedMySQL, MaterializedPostgreSQL, Dictionary, Replicated <other engines and links to the corresponding doc pages>.
```

Reply:
@sevirov, please, read the suggestion before committing it

Reply:
@tavplubix Yes, I've read it, I'm doing it now.